bee-network/bee-gossip/src/swarm/protocols/iota_gossip/io.rs

```diff
@@ -53,15 +53,13 @@ pub fn start_incoming_processor(
                 // from all once connected peers, hence if the following send fails, then it
                 // must be considered a bug.
 
-                // The remote peer dropped the connection.
-                internal_event_sender
-                    .send(InternalEvent::ProtocolDropped { peer_id })
-                    .expect("The service must not shutdown as long as there are gossip tasks running.");
-
                 break;
             }
         }
 
+        // Ignore send errors.
```

Comment:
Just curious: why ?

Reply:
we have a stronger guarantee here. It cannot panic. I changed it.